addons/web/static/src/views/fields/properties/property_value.scss

```diff
@@ -26,6 +26,14 @@ input.o_field_property_input[type=number] {
     }
 }
 
+.o_property_field_value {
+    .o_field_property_many2one_value img,
+    .o_m2m_avatar {
+        height: var(--Avatar-size, #{$o-avatar-size});
+        aspect-ratio: 1;
```

Comment:
The width: auto was removed, I'm guessing it was not necessary?

If so, would be nice to mention it in the commit message.

Reply:
yes, thanks to `aspect-ratio: 1;`, I mentionned it in the commit message